src/electron/npm/libreoffice.d.ts

```diff
@@ -0,0 +1,310 @@
+export namespace LibreOffice {
+  type ClipboardItem = {
+    mimeType: string;
+    buffer: ArrayBuffer;
+  };
+
+  type Rect = {
+    [name: string]: any;
+  };
+
+  type Size = {
+    [name: string]: any;
+  };
+
+  export interface DocumentClient {
+    /**
+     * @description add an event listener
+     * @param {string} eventName - the name of the event
+     * @param {() => void} callback - the callback function
```

Comment:
With TypeScript, you don't need to put the types in the JSDoc, just the type signature is fine. Also prevents having more than one source of truth.

Reply:
Ah you are right. I am used to using googles eslint rules for all my projects which fails if the jsdocs don't match the actual function methods. I will remove them here so as not to create extra work